src/tess_ephem/ephem.py

```diff
@@ -96,19 +96,140 @@ def predict_sky(self, time: Time) -> DataFrame:
+                )
+            # If object is not observed by TESS at time, skip time.
+            except ValueError:
+                continue
+
+        if len(df) == 0:
+            print("Warning: Target not observed by TESS at defined times.")
+        else:
+            # Make column names lowercase in df
+            df.columns = [x.lower() for x in df.columns]
+            # Make sector, camera, ccd integers
+            df = df.astype({"sector": int, "camera": int, "ccd": int})
+            if verbose:
+                df = df.merge(sky, on="time", how="inner")
+            df = df.set_index("time")
+            if not verbose:
+                df = df[["sector", "camera", "ccd", "column", "row"]]
+
         return df
 
+    @staticmethod
+    def from_sector(
+        target: str,
+        sector: int | None,
+        id_type: str = "smallbody",
+        step: str = "12H",
+        return_time: bool = False,
+    ):
+        """
+        Initialises TessEphem object from sector number.
+
+        Parameters
+        ----------
+        target : str
+            Horizons target ID.
+        sector : int or None
+            Sector number.
+        id_type : str
+            JPL/Horizons target identifier type.
+            One of "smallbody", "majorbody", "designation", "name", "asteroid_name",
+            "comet_name", or "designation".
+        step : str
+            Resolution at which ephemeris data will be obtained from JPL Horizons.
+        return_time: bool
+            If True then return start and stop time of sector.
+
+        Returns
+        -------
+        TessEphem
+            Initialiased TessEphem object.
+        start : Time
+            If return_time, sector start time.
+        stop : Time
+            If return_time, sector end time.
+        """
+
+        # If no sector passed, use all sectors in pointings file.
+        if sector is None:
+            # Define start and stop using pointings file from tesswcs
+            start, stop = Time(pointings["Start"][0], format="jd"), Time(
+                pointings["End"][-1], format="jd"
+            )
+
+        else:
+            # Sector must be in pointings
+            if sector not in pointings["Sector"]:
+                raise KeyError(
+                    "Sector must be in range {0}-{1}".format(
+                        pointings["Sector"][0], pointings["Sector"][-1]
+                    )
+                )
+
+            # Define start and stop of sector using pointings file from tesswcs
+            start, stop = Time(
+                pointings[pointings["Sector"] == sector]["Start"][0], format="jd"
+            ), Time(pointings[pointings["Sector"] == sector]["End"][0], format="jd")
+
+        # Buffer of one day on start, stop for future interpolation of ephemeris.
+        start_buffer, stop_buffer = start - TimeDelta(1, format="jd"), stop + TimeDelta(
+            1, format="jd"
+        )
```

Comment:
just for readability of the code, we could split this variable definitions per line:
```
start = ...
stop = ...
```
etc

Reply:
Done.